.github/no-response.yml

```diff
@@ -0,0 +1,10 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 14
+# Label requiring a response
+responseRequiredLabel: needs-more-info
+# Comment to post when closing an Issue for lack of response. Set to `false` to disable
+closeComment: >
+  This issue has been automatically closed due to no response from the original author.
```

Comment:
@mairaw Am I right to assume that the bot will only close the issue if the last person to respond was an MSFT user?

Reply:
What if the last person to respond is a non-MSFT user, but is helping?

Reply:
So the way it works is that the user has to respond. If they respond, then the label is automatically removed. We can keep an eye on issues being closed. But I find this easier than keeping an eye and checking when we applied the label. And if the author comments after it's closed, it automatically reopens too. You can see the ones that were closed in the docs repo with the label still on because the customer never replied: https://github.com/dotnet/docs/issues?q=is%3Aissue+label%3Aneeds-more-info+is%3Aclosed